model/jvm/src/main/resources/edit-metadata.txt

```diff
@@ -103,7 +103,7 @@ LAR,Quality,Q037,"If lien status = 2, then loan amount should be ≤ $250 ($250
 LAR,Quality,Q038,"If lien status = 3, then loan amount should be ≤ $100 ($100 thousand).","If the loan is not secured by a lien (lien status = 3), then the reported loan amount should be less than or equal to $100 thousand. Please verify and update your file if needed.",Lien Status; Loan Amount
 LAR,Quality,Q025,"If loan purpose = 1 and property type = 1, then loan amount should be > $10 thousand.","If the loan is reported as a home purchase (loan purpose = 1), one to four-family (property type =1), then the reported loan amount should be greater than $10 thousand. Please verify and update your file if needed.",Loan Amount; Loan Purpose; Property Type
 LAR,Quality,Q032,"If action taken type = 1, then action taken date should not equal the date application received.","If the loan was originated (action taken type = 1), then the reported action taken date should not equal the reported date application was received. Please verify and update your file if needed.",Type of Action Taken; Date of Action; Date Application Received
-LAR,Quality,Q068,"If action taken type = 1-5, 7 or 8, and applicant ethnicity, race and sex = 4,7,4 respectively, and co-applicant ethnicity, race and sex = 4,7,4, respectively, and there is no co-applicant, then co-applicant ethnicity, race, and sex should = 5,8,5, respectively.","The loan is reported as not purchased by your institution (action taken type = 1-5, 7 or 8), with no applicant (applicant ethnicity = 4, applicant race = 7, applicant sex = 4), and co-applicant as not applicable (co-applicant ethnicity = 4, co-applicant race = 7, co-applicant sex = 4). If there is no co-applicant on the loan, the co-applicant data fields should instead indicate that there is no co-applicant (co-applicant ethnicity = 5, co-applicant race = 8, co-applicant sex = 5). Please verify and update your file if needed. ",Type of Action Taken; Applicant Ethnicity; Applicant Race: 1; Applicant Sex; Co-applicant Ethnicity; Co-applicant Race: 1; Co-applicant Sex 
+LAR,Quality,Q068,"If action taken type = 1-5, 7 or 8, and applicant ethnicity, race and sex = 4,7,4 respectively, and co-applicant ethnicity, race and sex = 4,7,4, respectively, and there is no co-applicant, then co-applicant ethnicity, race, and sex should = 5,8,5, respectively.","The loan is reported as not purchased by your institution (action taken type = 1-5, 7 or 8), with no applicant (applicant ethnicity = 4, applicant race = 7, applicant sex = 4), and co-applicant as not applicable (co-applicant ethnicity = 4, co-applicant race = 7, co-applicant sex = 4). If there is no co-applicant on the loan, the co-applicant data fields should instead indicate that there is no co-applicant (co-applicant ethnicity = 5, co-applicant race = 8, co-applicant sex = 5). Please verify and update your file if needed. ",Type of Action Taken; Applicant Ethnicity; Applicant Race: 1; Applicant Sex; Co-applicant Ethnicity; Co-applicant Race: 1; Co-applicant Sex
```

Comment:
Ah, on second look, there's an extra space that was trimmed off the end of each of these lines.  Happened automatically on file save in IntelliJ.